widgets-intro.md

```diff
@@ -788,7 +788,7 @@ global keys because they can hold focus, which means they receive any text the
 user enters into the app. The
 [`Focus`](https://docs.flutter.io/flutter/widgets/Focus-class.html) widget
 keeps track of which
-[`InputState`](https://docs.flutter.io/flutter/material/InputState-class.html)
+[`State`](https://docs.flutter.io/flutter/material/State-class.html)
```

Comment:
`https://docs.flutter.io/flutter/widgets/State-class.html`

This edit is fine to fix the link, but this paragraph isn't very sensible anymore.  We do this automatically for developers now.  Maybe file a bug to revisit this paragraph?